### ISSUE TITLE
Plugin could not pass PluginDisableEvent

### ISSUE THREAD

**Issue by DrOreo002** [VolmitSoftware/React#473]
Full error log can be found at : https://pastebin.com/kMtqwCZm

## Reproducing
Run the server, and then stop it. And see.., it will freeze the server for well 5 or 6 seconds. Pretty annoying for a dev like me who test the plugin on localhost server, while having this plugin enabled for performance testing purpose

**Comment by Sikatsu** [comment]
This doesn't only happen with plugman. Normal server shutdown: https://pastebin.com/G0EKv7KS

**Comment by cyberpwnn** [comment]
```
14.02 09:34:58 [Server] INFO Disabling React v6.599
14.02 09:34:58 [Server] ERROR Could not pass event PluginDisableEvent to React v6.599
14.02 09:34:58 [Server] INFO org.bukkit.event.EventException: null
14.02 09:34:58 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:309) ~[my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500) [my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:485) [my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:358) [my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:422) [my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO at org.bukkit.plugin.SimplePluginManager.disablePlugins(SimplePluginManager.java:415) [my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO at org.bukkit.craftbukkit.v1_13_R2.CraftServer.disablePlugins(CraftServer.java:370) [my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.stop(MinecraftServer.java:621) [my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO at org.spigotmc.RestartCommand.restart(RestartCommand.java:119) [my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO at org.spigotmc.RestartCommand.restart(RestartCommand.java:40) [my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO at org.spigotmc.WatchdogThread.run(WatchdogThread.java:95) [my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO Caused by: java.util.NoSuchElementException: tiny-React
14.02 09:34:58 [Server] INFO at io.netty.channel.DefaultChannelPipeline.getContextOrDie(DefaultChannelPipeline.java:1119) ~[my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO at io.netty.channel.DefaultChannelPipeline.remove(DefaultChannelPipeline.java:451) ~[my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO at com.comphenix.protocol.injector.netty.PipelineProxy.remove(PipelineProxy.java:318) ~[?:?]
14.02 09:34:58 [Server] INFO at primal.bukkit.nms.TinyProtocol.close(TinyProtocol.java:204) ~[?:?]
14.02 09:34:58 [Server] INFO at primal.bukkit.nms.TinyProtocol.onPluginDisable(TinyProtocol.java:109) ~[?:?]
14.02 09:34:58 [Server] INFO at sun.reflect.GeneratedMethodAccessor869.invoke(Unknown Source) ~[?:?]
14.02 09:34:58 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_131]
14.02 09:34:58 [Server] INFO at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_131]
14.02 09:34:58 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:305) ~[my.jar:git-Spigot-b0f4c22-51c118b]
14.02 09:34:58 [Server] INFO ... 11 more
```

**Comment by cyberpwnn** [comment]
Fixed in 6.604
